src/Files.App/Views/Settings/AboutPage.xaml

```diff
@@ -15,7 +15,8 @@
 		<StackPanel
 			HorizontalAlignment="Stretch"
 			VerticalAlignment="Stretch"
-			Spacing="4">
+			Spacing="4"
+			Loading="{x:Bind ViewModel.About_Loading}">
```

Comment:
Instead of loading when the page loads, would it make sense to load it when the licenses section is expanded?

Reply:
Done.